### ISSUE TITLE
why create 168 excess bundles in webpack

### ISSUE THREAD

**Issue by zazzzf** [arkokoley/pdfvuer#32]
[image](https://user-images.githubusercontent.com/40158049/65141854-0fd8ab80-da44-11e9-948e-d4ee2dd5901e.png)

**Comment by Sylk** [comment]
Curious as to this too...

**Comment by Sylk** [comment]
Would really appreciate the answer before I abandon your package @arkokoley

**Comment by zazzzf** [comment]
I used the latest  vue-pdf without this problem.

**Comment by Sylk** [comment]
@zazzzf I've tried this in a new project, it still does it there. Have you changed some internal settings?

**Comment by arkokoley** [comment]
@zazzzf @Sylk This seems to be some default setting with Webpack. Never bothered looking too deep into this. Will take a look.

**Comment by Sylk** [comment]
@arkokoley thank you, give us an update if you find out what's going wrong. I appreciate the effort here.

**Comment by lobo-tuerto** [comment]
Observing the same behaviour in a new app.

**Comment by Sammyjo20** [comment]
This is the code splitting that has been built into it. Webpack will split Mozilla’s PDF interface to improve load times (I think)

**Comment by Sylk** [comment]
This looks horribly messy, is there no way to stop it?

**Comment by Sylk** [comment]
Would really appreciate a follow up  @arkokoley

**Comment by tartinesKiller** [comment]
This just seems to work as intended, concerning code-splitting. 10 chunks of 100kB are better than one 1MB package, especially if not all parts are used at the same time.
https://webpack.js.org/guides/code-splitting/

**Comment by halws** [comment]
There are a few lines which I've removed and then that bundles are gone.
![screen1](https://user-images.githubusercontent.com/16323637/70371409-e9077400-18d2-11ea-8771-04a3300d0692.png)

this code [start from 38 line](https://github.com/arkokoley/pdfvuer/blob/2cdb9b2d22c0f8e9bdc648cf176b404ae0476725/src/Pdfvuer.vue#L38)

I've tested  thas for a some time, and it's ok without that. So do we really need this?

**Comment by tartinesKiller** [comment]
More on cmaps: https://stackoverflow.com/questions/32764773/what-is-a-pdf-bcmap-file
For the pdf you've tested, theses files were not needed, and thus not downloaded by the client. But if at some point a PDF require these cmaps to be displayed, then the client will fetch'em. 
It's like you removed some files of your OS, everything could be fine until the OS needs one of those files.
It's up to you to choose if you want to take the risk of being screwed up one day for not including a few extra files.

**Comment by a-bashtannik** [comment]
The reason is [here](https://github.com/arkokoley/pdfvuer/blob/2cdb9b2d22c0f8e9bdc648cf176b404ae0476725/src/Pdfvuer.vue#L43): 
```
/* webpackChunkName: "noprefetch-[request]" */
```
Replace it with 
```
/* webpackChunkName: "pdfjs/noprefetch-[request]" */
```
And everything will be on `pdfjs/` path, but it doesn't resolve `pdfjs` output configuration problem, it still creates worker files on the root path, so we have no way to configure custom bundles. 

It's more about webpack configuration.

**Comment by markschultz** [comment]
can this be made optional? see [vue-pdf](https://github.com/FranckFreiburger/vue-pdf/commit/10a4e50f624c7d68ce876050af2c1319c0f084b5) for easy example.

**Comment by arkokoley** [comment]
@markschultz Thanks for the heads up on the commit. Will try to replicate that in pdfvuer.

**Comment by arkokoley** [comment]
Fixed this in v1.6.0

**Comment by pixelpaulaus** [comment]
strange, when my webpack builds this... the worker file  is built with just the '.js' extension. But the file requested in the browser is '.worker.js', so gets a 404 error.
